etc/home-env-funcs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+_sirepo_dev_mail=~/mail
```

Comment:
I think these should be globally available and overridable so that we can test the infrastructure:

`export SIREPO_DEV_MAIL_DIR=${SIREPO_DEV_MAIL_DIR:-$HOME/mail}`

Use `$HOME` so it works inside shifter.